app/scripts/models/reliers/oauth.js

```diff
@@ -32,6 +32,7 @@ define(function (require, exports, module) {
     code_challenge: Vat.codeChallenge().renameTo('codeChallenge'),
     code_challenge_method: Vat.codeChallengeMethod().renameTo('codeChallengeMethod'),
     keys_jwk: Vat.keysJwk().renameTo('keysJwk'),
+    login_hint: Vat.email(),
```

Comment:
For consistency, it's probably worth using `login_hint: Vat.email().renameTo('loginHint')` here, similar to what's done with `code_challenge_method` and `keys_jwk` above.